package.json

```diff
@@ -91,48 +92,66 @@
   ],
   "license": "MIT",
   "peerDependencies": {
+    "react": "*",
     "react-native": "*"
   },
   "devDependencies": {
-    "@react-native-community/eslint-config": "^3.0.1",
+    "@react-native/eslint-config": "^0.73.1",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/commit-analyzer": "^9.0.2",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^8.0.2",
     "@semantic-release/npm": "^9.0.1",
     "@semantic-release/release-notes-generator": "^10.0.3",
     "@testing-library/react-hooks": "^7.0.2",
-    "@types/jest": "^27.4.1",
-    "@types/react": "17.0.39",
-    "@types/react-native": "^0.67.2",
-    "eslint": "^7",
-    "eslint-plugin-prettier": "^4.0.0",
+    "@types/jest": "^29.5.5",
+    "@types/react": "16.9.0",
```

Comment:
Why did you lower the `@types/react` version?

Reply:
Not sure 😅